x-pack/test/functional/apps/maps/joins.js

```diff
@@ -78,15 +79,21 @@ export default function ({ getPageObjects, getService }) {
       const layersForVectorSource = mapboxStyle.layers.filter(mbLayer => {
         return mbLayer.id.startsWith(VECTOR_SOURCE_ID);
       });
+      // Color is dynamically obtained from eui source lib
+      const dynamicColor = layersForVectorSource[0].paint['circle-stroke-color'];
```

Comment:
Why do you have to change the color in the expect statements? The same color should be used every time since the index of the color palettes should always be the same and thus return the same color

Reply:
> The same color should be used every time since the index of the color palettes should always be the same and thus return the same color

Not necessarily. See the conversation above with @cchaos about the colors potentially getting out of sync if/when there are palette updates. We don't really want to hardcode the color and have that fragility, but we can't easily obtain the color dynamically either.